script/dev.sh

```diff
@@ -0,0 +1 @@
+fd '\.(mjs|jsx)$' | entr -c node test/tests.mjs
```

Comment:
I like better using simple tools in combination than relying on big frameworks for watching files